hxmath/math/Vector2.hx

```diff
@@ -726,17 +726,23 @@ abstract Vector2(Vector2Type) from Vector2Type to Vector2Type
      * @param pivot     The pivot point to rotate around.
      * @return          The modified object.
      */
-    public inline function rotate(angle:Float, pivot:Vector2):Vector2
+    public inline function rotate(angle:Float, ?pivot:Vector2):Vector2
     {
         var self:Vector2 = this;
         
-        var cosAngle = Math.cos(angle);
-        var sinAngle = Math.sin(angle);
-        var dx = self.x - pivot.x;
-        var dy = self.y - pivot.y;
+        var cos = Math.cos(angle);
```

Comment:
Minor (and OCD 😄): can you keep the cosAngle/sinAngle name? That way it'll be consistent with the other code.

Reply:
Totally understand haha. Fixed.